src/query/expression/tests/it/types.rs

```diff
@@ -35,6 +37,36 @@ fn test_timestamp_to_string_formats() {
     );
 }
 
+#[test]
+fn test_parse_jiff() {
+    let (mut tm, offset) = BrokenDownTime::parse_prefix(
```

Comment:
Maybe can consider https://docs.rs/jiff/latest/jiff/fmt/strtime/fn.format.html it will be better if format is valid.

Reply:
but this's just for testing..